spec/system/admin_views_dashboard_spec.rb

```diff
@@ -46,17 +48,16 @@
     end
   end
 
-  it "can see the last case contact and navigate to it" do
+  it "can see the last case contact and navigate to it", js: false do
     volunteer = create(:volunteer, :with_case_contact_wants_driving_reimbursement, email: "casa@example.com")
     sign_in admin
 
     visit root_path
 
-    expect(page).to have_text(volunteer.most_recent_contact.occurred_at.strftime("%B %-e, %Y"))
-    expect(page).to have_text("20") # miles driven
+    expect(page).to have_text("August 29, 2020")
 
     within "#volunteers" do
-      click_on volunteer.most_recent_contact.occurred_at.strftime("%B %-e, %Y")
+      click_on "August 29, 2020"
```

Comment:
This seems date-based and likely to fail in the future

Reply:
I used the time travel code: `travel_to Time.zone.local(2020,8,29,4,5,6)`, so the date will be fixed. There had been a comment on a similar spec that it had been failing locally due to timezone issues, but then I saw [this commit](731f7283) and it seems like maybe the time zone issue had already been fixed. So I can also revert this.

Reply:
Ah I see it now. Sorry about that, my mistake